opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/jdbc/JdbcTracingTest.java

```diff
@@ -0,0 +1,56 @@
+package io.opentracing.contrib.spring.cloud.jdbc;
+
+import io.opentracing.contrib.jdbc.TracingConnection;
+import io.opentracing.contrib.spring.cloud.MockTracingConfiguration;
+import io.opentracing.mock.MockTracer;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import javax.sql.DataSource;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+@SpringBootTest(classes = {MockTracingConfiguration.class, JdbcAutoConfiguration.class})
+@RunWith(SpringJUnit4ClassRunner.class)
+public class JdbcTracingTest {
```

Comment:
This needs a _wiring_ test. e.g. call database in the rest controller and assert that server span and db span are in the same trace.